app-arch/lz4/lz4-1.9.3-r2.ebuild

```diff
@@ -14,7 +14,7 @@ LICENSE="BSD-2 GPL-2"
 # https://abi-laboratory.pro/tracker/timeline/lz4/
 SLOT="0/r132"
 KEYWORDS="~alpha amd64 arm arm64 hppa ~ia64 ~m68k ~mips ppc ppc64 ~riscv ~s390 sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
-IUSE="static-libs"
+IUSE="lib-only static-libs"
```

Comment:
I'd consider `IUSE=minimal` instead, as that's an existing global use flag used by many packages.

Reply:
I'm okay with `minimal` as well